multirun.bzl

```diff
@@ -105,6 +105,15 @@ multirun = rule(
             cfg = "exec",
             executable = True,
         ),
-    },
-    executable = True,
-)
+    }
+
+    if allowlist:
+        attrs["_allowlist_function_transition"] =  attr.label(default = allowlist)
```

Comment:
do we need to pass this or can we just always set this even if it's not used? (maybe there's a downside to that though?)

Reply:
* for configurations with a state transition and _without_ `_allowlist_function_transition ` attribute Bazel complains
```
ERROR: .../rules_multirun/multirun.bzl:113:16: Use of Starlark transition without allowlist attribute '_allowlist_function_transition'. See Starlark transitions documentation for details and usage: //deploy:aws.bzl NORMAL
```
which is a TODO to be removed https://github.com/bazelbuild/bazel/blob/e2189245/src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkRuleClassFunctions.java#L923-L935

* for "exec" and "target" configuration and _with_ _allowlist_function_transition ` attribute Bazel complains
```
ERROR: .../rules_multirun/multirun.bzl:113:16: Unused function-based split transition allowlist: //deploy:aws.bzl NORMAL
```
in the else branch at https://github.com/bazelbuild/bazel/blob/e2189245/src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkRuleClassFunctions.java#L937-L944

>  just always set this even if it's not used
it is not possible to set the attribute unconditionally.